sqlglot/dataframe/sql/readwriter.py

```diff
@@ -26,7 +26,7 @@ def table(self, tableName: str) -> DataFrame:
             .from_(tableName)
             .select(
                 *(
-                    column if should_identify(column, "safe") else f'"{column}"'
+                    column if can_identify(column, "safe") else f'"{column}"'
```

Comment:
I renamed `should_identify` to `can_identify` exactly because this section was awkward to read. I think it's easy to read this and go "why are we _identifying_ if we should _not_ identify, according to the condition?".